src/gcc/index.md

```diff
@@ -3,6 +3,7 @@ title: Galaxy Community Conferences (GCCs)
 ---
 
 <div class="text-center">
+
```

Comment:
@tnabtaf Marked got more strict in its parsing and these are required now.   Unfortunately they're probably scattered around the hub, so if you see un-parsed markdown inside html, this is the fix.

Reply:
Crap, crap, crap.  

Writing a parser to catch this will be harder, as HTML can span multiple lines, and / or have markdown on the same line.  And it probably applies to all HTML?